.github/workflows/mirror-calico.yaml

```diff
@@ -4,9 +4,18 @@ on:
     # run every 12h
     - cron:  '0 */12 * * *'
```

Comment:
One thing I'm wondering about is if these actions should really be called twice a day. Maybe once a week would be enough?

Reply:
I probably miss something, but wouldn't the test just grab whatever is the latest docker package available in flatcar GHCR? Or is there somewhere a code that is sensitive to version mismatches between our calico docker images and actual newest upstream release?

Now I'm wondering if those sync actions should rather reside in mantle repo. We could merge calico docker image sync together with the calico manifest updates.

Reply:
Hm, I did a bit of digging here and I think our issue used to be that we were pulling the tigera operator manifest from a link like `https://docs.projectcalico.org/manifests/tigera-operator.yaml`. By the looks of it, it was always pointing to the latest release of calico. So there was certainly a window where the just released tigera-operator was trying to pull in newer versions of calico images that weren't yet mirrorred. Now, we are using a versioned link (like `https://raw.githubusercontent.com/projectcalico/calico/v3.24.0/manifests/tigera-operator.yaml`), so I think that as long as the version in the link and the version of our mirrored docker images are in sync, we should be safe. And we probably could then just run the action once a week.

Anyway, I think I'll move these actions to mantle. They are not related to anything in the scripts repo.